.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -13,18 +13,21 @@ body:
         - bug
         - batch
         - config
-        - flepiconfig
+        - dependency
         - documentation
         - docstring
         - duplicate
         - enhancement
+        - flepicommon
+        - flepiconfig
         - gempyor
         - good first issue
         - inference
```

Comment:
Only comment I have is that I think there was prior discussion (see https://github.com/HopkinsIDD/flepiMoP/pull/327#discussion_r1784892799) on "inference" referring to the concept vs the R package. what does it refer to here?

Reply:
oh that's a good point. I think i'll add an `r-` prefix to those that refer to r packages. Then I suppose i'll add a separate tag of just `inference` to refer to the inference framework.